### ISSUE TITLE
Question regarding the point masks

### ISSUE THREAD

**Issue by yunjinli** [facebookresearch/ShapeR#16]
Dear author,

thank you so much for open-sourcing this amazing work!!

I just have a small question regarding the sensitivity of the point mask, in the paper, it is shown that the point mask is derived from SLAM points which come from high gradient regions of the images (edges, corners, ...), the intuition is also quite straighforward that for multi-modal inputs you want to give dino sort of an area of interest for generation. 

My question is: Have you tried to feed dense point cloud -> sparsify it -> project for point mask? Because in this way then you are not feeding the high gradient regions into the images and I am just wondering if that would degrade the performance. (sort of like the current DA3 workaround you implemented but for more complicated scene)

Thank you so much and I look forward to your reply :)

Best,

Jim

**Comment by nihalsid** [comment]
Hey Jim! Thanks for your interest in the work and your question. Sorry for the late response!

I think this can work much like the DA3 workaround. However, there are two things why this would cause a degradation:

1) If you're thinking of passing the dense point cloud for scenes rather than objects, it wont work well out of the box, since we trained entirely on single objects
2) The dense depth leads to some degradation since we only trained with semidense, sparse aria like point clouds. If you want the best performance on dense depth, best finetune the model on some dense depth data.
